### ISSUE TITLE
Training only on a subset of classes present in TFRecords

### ISSUE THREAD

**Issue by ujjwal-ai** [tensorflow/models#9375]
There appears to be no clarity or documentation on the following issue:

Suppose one wants to train a person detector using COCO dataset. The TFRecords for COCO normally contain all object classes. Though one can create a separate set of TFRecords for persons only, is there any feature in Object Detection API where one can fine-tune only for specific classes ( counting all other categories as background ) ?

**Comment by SamLee514** [comment]
If you're still having this issue, you could try just customizing the label map file so that it only contains the classes that are relevant to your project (and adjust the number of classes in the pipeline.config file accordingly). I haven't tried this myself, but it's where I would start if I had this problem!

**Comment by divyanshu-quantiphi** [comment]
Did we find a better solution?
